### ISSUE TITLE
Sorting of tags by name in alphabetically order

### ISSUE THREAD

**Issue by stefan38** [tagspaces/tagspaces#869]
It would be nice if we could sort tags assigned to files by name in alphabetically order (based on tag groups in case we have tags from multiple tag groups assigned to a file) as we can do inside tag groups (in tag library).

**Comment by roxfanuk** [comment]
I agree. In fact it might be nice to have this as default behaviour, that file tags are in alphanumerical order
